dask/dataframe/io/parquet/arrow.py

```diff
@@ -839,6 +840,10 @@ def write_partition(
             index_cols = []
 
         t = cls._pandas_to_arrow_table(df, preserve_index=preserve_index, schema=schema)
+        if custom_metadata:
+            _md = t.schema.metadata
+            _md.update(custom_metadata)
+            t = t.replace_schema_metadata(metadata=_md)
```

Comment:
@jorisvandenbossche - I saw that this method is labeled as "Experimental," but it has been around for years now (as far as I can tell).  Is there any reason not to use it here?

Reply:
I think that's fine to use (we should update the docstring)

Reply:
-> https://issues.apache.org/jira/browse/ARROW-11980